troubleshoot/ports/troubleshoot_tcp.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package ports
+
+import (
+	"fmt"
+	"net"
+	"time"
+)
+
+type troubleShootTcp struct {
+}
+
+func (tcp *troubleShootTcp) dailPort(hostPort *hostPort) string {
```

Comment:
**suggestion:** Should this be return a (string, error) so that a proper error can be checked passed through?

Reply:
I believe returning only error will suffice.